### PR TITLE
Add Solar theme

### PR DESCRIPTION
**Pull request by tarvos21**
- solar.theme.config
- LESS source code
- prebuilt CSS files(update to Semantic-UI 2.2.10)

Main principal here is to use `{component}.overrides` as little as possible, best to make all definitions in the `{component}.varaibles`.

Obvious problems left:

- default square checkbox: can not be checked
- star rating icon: rating can not be themed, or the star rating icon will disappear(no content at all)

These two problems exist in any theme that adds a theme other than default to checkbox and rating.

### CODE REVIEW

File: src/themes/bootswatch/solar/modules/sidebar.varibles [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (xiaohanyu) [inline src/themes/bootswatch/solar/modules/sidebar.varibles]: ? one line diff?

Comment (tarvos21) [inline src/themes/bootswatch/solar/modules/sidebar.varibles]: Oh, sorry for this. I will check it again.

**Comment by tarvos21** [comment]
OK, I will update Semantic-UI.

**Comment by tarvos21** [comment]
Updated to Semantic-UI 2.2.10 through `npm install semantic-ui --save` command, then `gulp build`.

**Pull request merged**